README.md

```diff
@@ -1,5 +1,7 @@
 # MageForge for Magento 2
 
+[![Codacy Badge](https://app.codacy.com/project/badge/Grade/365d32460022451ba3918f5643341753)](https://app.codacy.com/gh/dermatz/mageforge/dashboard?utm_source=gh&utm_medium=referral&utm_content=&utm_campaign=Badge_grade)
+
 ![Mageforge Hero](./.github/assets/mageforge-hero.jpg)
 
```

Comment:
Is it more pleasing below the hero banner?

<img width="1108" alt="image" src="https://github.com/user-attachments/assets/8ba2735d-d3b2-4672-8fce-be9d286e5593">

If you say no it's fine, just asking questions 😅

Reply:
```suggestion
![Mageforge Hero](./.github/assets/mageforge-hero.jpg)

[![Codacy Badge](https://app.codacy.com/project/badge/Grade/365d32460022451ba3918f5643341753)](https://app.codacy.com/gh/dermatz/mageforge/dashboard?utm_source=gh&utm_medium=referral&utm_content=&utm_campaign=Badge_grade)
```